# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/qt-advanced-docking-system/hotfix-module-path.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

CMake Error at src/CMakeLists.txt:4 (include):
  include could not find requested file:

    Versioning

--- Qt-Advanced-Docking-System-4.5.0/CMakeLists.txt.vanilla	2026-02-08 18:12:15.638887759 +0100
+++ Qt-Advanced-Docking-System-4.5.0/CMakeLists.txt	2026-02-08 18:11:48.052509169 +0100
@@ -8,8 +8,8 @@
 # we can override this behavior by explicitly setting ADS_VERSION and
 # skipping the git checks. This is useful for cases where this project is being
 # used independently of its original git repo (e.g. vendored in another project)
+set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules ${CMAKE_MODULE_PATH})
 if(NOT ADS_VERSION)
-    set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules ${CMAKE_MODULE_PATH})
     include(GetGitRevisionDescription)
     git_describe(GitTagVersion --tags)
     string(REGEX REPLACE "^([0-9]+)\\..*" "\\1" VERSION_MAJOR "${GitTagVersion}")
